src/factories/nodeFlowRun.ts

```diff
@@ -130,10 +130,10 @@ export async function flowRunContainerFactory(node: RunGraphNode) {
 
     const labelMinLeft = Math.max(barRight, buttonRight)
     const inside = barWithoutMargin > labelMinLeft + label.width
-    const y = bar.height / 2
+    const yOffset = 0.5
+    const y = bar.height / 2 - label.height / 2 - yOffset
     const x = inside ? labelMinLeft + margin : arrowButton.x + arrowButton.width + margin
 
-    label.anchor.set(0, 0.5)
```

Comment:
I couldn't quite understand where this spacing was coming from, but it performed better as an offset on the y for some reason.

Reply:
yeah I was trying to center by setting the anchor to 0.5 and then positioning to bar.height / 2. But with a predictable line height this probably doesn't work quite right. (Wasn't quite right to begin with).